playbooks/cluster/kubernetes/config.yml

```diff
@@ -7,14 +7,14 @@
       with_items:
         - "{{ playbook_dir }}/vars/default_vars.yml"
   roles:
-    - kubernetes-prerequisites
+    - "{{ playbook_dir }}/roles/kubernetes-prerequisites"
 
 - hosts: masters
   remote_user: root
   roles:
-    - kubernetes-master
+    - "{{ playbook_dir }}/roles/kubernetes-master"
```

Comment:
I think since we have roles symlink in playbook's directory, we can drop all these `{{ playbook_dir }}/roles/` part from role name.